orc8r/tools/packer/scripts/cleanup.sh

```diff
@@ -13,19 +13,13 @@
 ################################################################################
 
 # Uninstall Ansible and remove PPA.
-apt -y remove --purge ansible
+apt-get -y remove --purge ansible
 apt-add-repository --remove ppa:ansible/ansible
 
 # Apt cleanup.
-apt autoremove
-apt update
+apt-get -y autoremove
+apt-get -y autoclean
+apt-get update
 
 # Delete unneeded files.
 rm -f /home/vagrant/*.sh
-
-# Zero out the rest of the free space using dd, then delete the written file.
```

Comment:
A similar thing is now done in `minimize_ubuntu.sh`

Reply:
Small "changelog":

- The change from apt to apt-get was made to satisfy an Ansible warning about apt not having a stable CLI interface (see [here](https://askubuntu.com/questions/990823/apt-gives-unstable-cli-interface-warning))

-     The dd command was removed because it didn't work. A very similar deletion step is executed successfully in the 3rd party minimize_ubuntu.sh script.